ios-baseUITests/ios_baseUITests.swift

```diff
@@ -159,4 +109,57 @@ class ios_baseUITests: XCTestCase {
     
     XCTAssert(signInButton.isEnabled)
   }
+  
+  // swiftlint:disable line_length
+  /// These tests won't work because we need to mock headers and Swifter currently does not support this
```

Comment:
why are headers needed to mock the network call? This wasn't needed before if I remember correctly

Reply:
This is related to how are we saving session here 

`  func saveUserSession(
    _ user: User?,
    headers: [AnyHashable: Any]
  ) -> Bool {
    userDataManager.currentUser = user
    guard let session = Session(headers: headers) else { return false }
    sessionManager.saveUser(session: session)
    return userDataManager.currentUser != nil && sessionManager.currentSession?.isValid ?? false
  }
`

Reply:
And swifter doesn't support mocking header right now so that's why I couldn't fix this test